modules/sno-du-managing-network-secrets.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-managing-network-secrets_{context}"]
+= Managing network secrets
+
+To ensure that transactions with the single node are secure, you need to provide a pull secret.
```

Comment:
The ClusterDeployment CR provides some of the cluster installation attributes including the cluster name and domain (together these make up the FQDN of the cluster) and references to the AgentClusterInstall and BareMetalHost CRs. The pull secret is used to obtain necessary installation artifacts.

Reply:
added.